### PR TITLE
Chapter9

### PR DESCRIPTION
**Pull request by nazihaltajar**

### CODE REVIEW

File: Chapter9/Chapter9/RestaurantTableViewCell.swift [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+//
+
+import UIKit
+
+class RestaurantTableViewCell: UITableViewCell {
+    @IBOutlet var nameLabel: UILabel!
+    @IBOutlet var locationLabel: UILabel!
+    @IBOutlet var typeLabel: UILabel!
+    @IBOutlet var thumbnailImageView: UIImageView!
+    override func awakeFromNib() {
```
Comment (alexbartisro) [inline Chapter9/Chapter9/RestaurantTableViewCell.swift:17]: It's nice to have newlines between code blocks

Comment (alexbartisro) [inline Chapter9/Chapter9/RestaurantTableViewCell.swift:17]: This doesn't do anything, it's not needed

File: Chapter9/Chapter9/RestaurantTableViewCell.swift [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        super.awakeFromNib()
+    }
+
+    override func setSelected(_ selected: Bool, animated: Bool) {
```
Comment (alexbartisro) [inline Chapter9/Chapter9/RestaurantTableViewCell.swift]: Same

File: Chapter9/Chapter9/RestaurantTableViewController.swift [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+import UIKit
+
+class RestaurantTableViewController: UITableViewController {
+    var restaurantNames =
+        ["Cafe Deadend", "Homei", "Teakha", "Cafe Loisl", "Petite Oyster", "For Kee Restaurant",
+        "Po's Atelier", "Bourke Street Bakery", "Haigh's Chocolate", "Palomino Espresso", "Upstate",
+        "Traif", "Graham Avenue Meats And Deli", "Waffle & Wolf", "Five Leaves", "Cafe Lore", "Confessional",
+        "Barrafina", "Donostia", "Royal Oak", "CASK Pub and Kitchen"]
+
+    override func viewDidLoad() {
```
Comment (alexbartisro) [inline Chapter9/Chapter9/RestaurantTableViewController.swift]: This doesn't do anything, it's not needed

File: Chapter9/Chapter9/RestaurantTableViewController.swift [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+        super.viewDidLoad()
+    }
+
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        return 1
+    }
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
```
Comment (alexbartisro) [inline Chapter9/Chapter9/RestaurantTableViewController.swift:22]: Missing new line between code blocks

File: Chapter9/Chapter9/RestaurantTableViewController.swift [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+        cell.nameLabel.text = restaurantNames[indexPath.row]
+        cell.thumbnailImageView.image = UIImage(named: restaurantNames[indexPath.row])
+
+        return cell
+    }
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return restaurantNames.count
+    }
+
```
Comment (alexbartisro) [inline Chapter9/Chapter9/RestaurantTableViewController.swift]: Extra new line

**Review by alexbartisro — CHANGES_REQUESTED** [review]
Please address remarks first

File: Chapter9/Chapter9/RestaurantTableViewCell.swift [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+//  Chapter9
+//
+//  Created by Nazih Al Tajar on 15/03/2019.
+//  Copyright © 2019 sss. All rights reserved.
+//
+
+import UIKit
+
+class RestaurantTableViewCell: UITableViewCell {
+    @IBOutlet var nameLabel: UILabel!
```
Comment (alexbartisro) [inline Chapter9/Chapter9/RestaurantTableViewCell.swift:12]: It's good practice to make outlets private, in order not to be accessed directly

**Review by alexbartisro — CHANGES_REQUESTED** [review]

**Review by alexbartisro — APPROVED** [review]

File: Chapter9/Chapter9/RestaurantTableViewController.swift [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+        "Po's Atelier", "Bourke Street Bakery", "Haigh's Chocolate", "Palomino Espresso", "Upstate",
+        "Traif", "Graham Avenue Meats And Deli", "Waffle & Wolf", "Five Leaves", "Cafe Lore", "Confessional",
+        "Barrafina", "Donostia", "Royal Oak", "CASK Pub and Kitchen"]
+
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        return 1
+    }
+
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cellIdentifier = "datacell"
```
Comment (adinas23) [inline Chapter9/Chapter9/RestaurantTableViewController.swift]: Rename it to ```dataCellIdentifier```

**Pull request merged**